flytekit/__init__.py

```diff
@@ -1,4 +1,4 @@
 from __future__ import absolute_import
 import flytekit.plugins
 
-__version__ = '0.7.0b1'
+__version__ = '0.7.1'
```

Comment:
Please let me know which version I should be bumping this to

Reply:
what's the timeline for this?  how soon do you need it?